persistence/sql.go

```diff
@@ -0,0 +1,48 @@
+package persistence
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v4"
+	"log"
+	"math/rand"
+	"os"
+	"time"
+)
+
+const (
+	PostgresSchemaEnvVar = "POSTGRES_SCHEMA"
+
+	CreateSchemaIfNotExists = "CREATE SCHEMA IF NOT EXISTS"
+	SetSearchPathTo         = "SET search_path TO"
+	CreateTableIfNotExists  = "CREATE TABLE IF NOT EXISTS"
+	TableName               = "users"
+	TableSchema             = "(id int)"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func connectAndInitializeDatabase(postgresUrl string) error {
```

Comment:
Part of this (e.g. the conenction) is "production code", but the user piece is more "placeholder".

I think we should split it and add a comment that the user table piece is just a placerholder.

Reply:
Separate issue?